--- bin/installDeps.sh	2014-09-06 18:37:09.000000000 +0200
+++ bin/installDeps.sh.b	2014-09-27 21:08:33.997694886 +0200
@@ -36,8 +36,9 @@
 
 #check npm version
 NPM_VERSION=$(npm --version)
-if [ ! $(echo $NPM_VERSION | cut -d "." -f 1) = "1" ]; then
-  echo "You're running a wrong version of npm, you're using $NPM_VERSION, we need 1.x" >&2
+NPM_MAIN_VERSION=$(echo $NPM_VERSION | cut -d "." -f 1)
+if [ $(echo $NPM_MAIN_VERSION) = "0" ]; then
+  echo "You're running a wrong version of npm, you're using $NPM_VERSION, we need 1.x or higher" >&2
   exit 1 
 fi
 
